lib/helpers.js

```diff
@@ -36,7 +36,7 @@ export function childrenAsString(children?: string | Array<string>) {
 
 export function extractOpacity({visible, opacity}: OpacityProps) {
   // TODO: visible === false should also have no hit detection
-  if (visible === false) {
+  if (!visible || visible === false) {
```

Comment:
I think we should set opaque to 0 only if user explicitly passes `visible={false}`. All components that use this helper function don't have visible set by default, so it would break current behavior.